docs/HowToFAQ/FAQParallel.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Does Dafny verify methods in parallel?
+---
+
+## Question
+
+Does Dafny verify methods in parallel?
+
+## Answer
+
+For the most part, Dafny verifies each method in each module sequentially
+and attempts to prove all the assertions in a method in one go.
```

Comment:
On the CLI Dafny runs sequentially by default, but in the IDE the default is concurrent execution.

Reply:
Revised